client/js/renderPreview.js

```diff
@@ -12,6 +12,19 @@ function renderPreview(preview, msg) {
 		return;
 	}
 
+	preview.link = Handlebars.Utils.escapeExpression(preview.link);
```

Comment:
Prefetching https://msdn.microsoft.com/en-us/library/windows/desktop/ms644989(v=vs.85).aspx (which requires escaping according to our `parse` tests) is broken on this PR. Pretty this is the cause.